R/dataframe__frame.R

```diff
@@ -919,14 +907,20 @@ DataFrame_to_list = function(unnest_structs = TRUE) {
 #' df2 = pl$DataFrame(y = 1:4)
 #' df1$join(other = df2, how = "cross")
 DataFrame_join = function(
-    other, # : LazyFrame or DataFrame,
-    left_on = NULL, # : str | pli.RPolarsExpr | Sequence[str | pli.RPolarsExpr] | None = None,
-    right_on = NULL, # : str | pli.RPolarsExpr | Sequence[str | pli.RPolarsExpr] | None = None,
-    on = NULL, # : str | pli.RPolarsExpr | Sequence[str | pli.RPolarsExpr] | None = None,
-    how = c("inner", "left", "outer", "semi", "anti", "cross"),
+    other,
+    on = NULL,
+    how = c("inner", "left", "outer", "semi", "anti", "cross", "outer_coalesce"),
+    left_on = NULL,
```

Comment:
```suggestion
    how = c("inner", "left", "outer", "semi", "anti", "cross", "outer_coalesce"),
    ...,
    left_on = NULL,
```

https://github.com/pola-rs/polars/blob/c9a14687acf49019b755d8e2141917766bbd5eae/py-polars/polars/dataframe/frame.py#L6234-L6245

Reply:
Done, thanks